hijack/__init__.py

```diff
@@ -1,4 +1 @@
-# -*- coding: utf-8 -*-
-__version__ = '2.1.10'  # pragma: no cover
-
```

Comment:
I have little experience with setuptools_scm, so I do not quite understand how it will work.
What will we need to do to release a patch version, minor or major?

Reply:
Simple, you just create a GitHub release with the tag `2.2.4` or `3.0a1`.

You can also release a development version from any branch. That will result in a combination of the last tag and a dev postfix, like `2.2.4.dev2` (depending on how many commits you are behind the last tag).

Reply:
Thank you, everything is clear.